portal-impl/src/com/liferay/portal/servlet/filters/authverifier/AuthVerifierFilter.java

```diff
@@ -202,6 +220,61 @@ else if (_guestAllowed || (state == AuthVerifierResult.State.SUCCESS)) {
 		}
 	}
 
+	private List<AuthVerifierConfiguration> _buildAuthVerifierConfigurations(
+		Map<String, Object> initParametersMap) {
+
+		Map<String, Integer> authVerifierConfigurationIndexs = new HashMap<>();
```

Comment:
@csierra this map has never been added with anything.

Reply:
you are right... I actually get a warning from the IDE but didn't spot it... @arthurchan35 would you mind having a look?

Reply:
Looking